EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+india.com##.budgetbgimg
```

Comment:
This is not an ad. Move it to the AdGuard Annoyances -> `annoyances.txt` at the top with task link.

Reply:
Please check.